pom.xml

```diff
@@ -49,5 +49,6 @@
   <modules>
     <module>api</module>
     <module>wiki</module>
+    <module>webjar</module>
```

Comment:
The best practice is to list them in alphabetical order (i.e. match the order in the file system).